### ISSUE TITLE
Temporal Fusion Transformers - Val Losses don't improve at all

### ISSUE THREAD

**Issue by realiti4** [google-research/google-research#904]
I'm experimenting with the 4 dataset in the original paper. One thing that I noticed is that val losses usually gets it's best value in first couple of epochs and don't improve or stays the same. Is this normal? What is the point of training more than 2-3 epochs than? Also predefined eary calls usually stops the network between 10-20 epochs. The paper says that the model is trained for 6 hours on a V100 for electiricty dataset. Is there a reason why I am not seeing val loss improvements, or this is normal and that 6 hours training time is with hparam search included. I'd appreciate if you help me understand, thanks.
